src/lib/core/a11y/list-key-manager.ts

```diff
@@ -32,10 +32,18 @@ export class ListKeyManager {
     return this;
   }
 
-  /** Sets the focus of the list to the item at the index specified. */
-  setFocus(index: number): void {
+  /**
+   * Sets the focus of the list to the item at the index specified.
+   *
+   * @param index The index of the item to be focused.
+   * @param focusElement Whether or not to focus the element as well. Defaults to `true`.
+   */
+  setFocus(index: number, focusElement = true): void {
```

Comment:
I'm not sure it is a no-op. I think it was emitting the events twice.

Also, now that I dug into the code, I'm not sure we *want* to change this. The majority of places assume it does both; my chips is a special case where we don't want to focus it (because it happens on click; so it already has focus).

Separating this forces users to call two functions the majority of the time which feels odd.